docs/framework/configure-apps/file-schema/windows-workflow-foundation/system-servicemodel-of-workflow.md

```diff
@@ -102,4 +102,4 @@ This configuration section contains all the workflow configuration elements.
   
```

Comment:
It would be useful to add the configuration hierarchy here as well, even though it just has two entries (\<configuration> and \<system.serviceModel>). It's absence seems somewhat glaring.

Reply:
Just inserted it at usual place